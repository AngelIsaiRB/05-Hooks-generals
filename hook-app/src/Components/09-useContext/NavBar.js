import React from 'react'
import { Link, NavLink } from 'react-router-dom'

export const NavBar = () => {
    return (
        <nav className="navbar navbar-expand-sm navbar-dark bg-dark">
            <Link className="navbar-brand" to="./" > Usecontext </Link>
            <div className="collapse navbar-collapse" id="navbarNavAltMarkup">
                <div className="navbar-nav">
                    <NavLink exact activeClassName="active" className="nav-item nav-link " to="./">Home</NavLink >
                    <NavLink exact activeClassName="active" className="nav-item nav-link " to="./about">About</NavLink >
                    <NavLink exact activeClassName="active" className="nav-item nav-link " to="./login">Login</NavLink >
                   
                </div>

            </div>

        </nav>



        // <nav>
        //     <ul>
        //         <li> <Link to="./">Home </Link></li>
        //         <li><Link to="./about">about</Link></li>
        //         <li><Link to="./login">login</Link></li>
                
        //     </ul>
        // </nav>
    )
}
