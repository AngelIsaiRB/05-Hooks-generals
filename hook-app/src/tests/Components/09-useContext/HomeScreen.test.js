import { mount, shallow } from 'enzyme';
import React from 'react';
import { HomeScreen } from '../../../Components/09-useContext/HomeScreen';
import { UserContext } from '../../../Components/09-useContext/UserContext';

describe('Pruebas en <HomeScreen/>', () => {
    const user={
        name: 'John',
        email: 'john@example.com'
    }
    const wrapper =mount(
    <UserContext.Provider value={{user}} >
        <HomeScreen />
    </UserContext.Provider>
    );

    test('should de mostrase correctamente', () => {
        expect(wrapper).toMatchSnapshot()
    });
    
})
