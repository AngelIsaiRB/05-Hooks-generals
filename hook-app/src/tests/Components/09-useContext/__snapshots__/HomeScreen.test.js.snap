// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HomeScreen/> should de mostrase correctamente 1`] = `
<div>
  <h1>
    Home Screen
  </h1>
  <hr />
  <pre>
    {
   "name": "John",
   "email": "john@example.com"
}
  </pre>
</div>
`;
